doc/development.md

```diff
@@ -0,0 +1,133 @@
+# Installing the OpenQuake Hazardlib for development
+
```

Comment:
I guess this should be renamed Installing the `OpenQuake Engine for development`. There should be a single document both for engine and hazardlib.